src/service/templates/service.model.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = function(options) {
+  // instantiate and return model adapter of choice
+}
```

Comment:
Agreed. We have some stuff that we need to talk to the @feathersjs/core-team about regarding these models.